libraries/testAutomation/es-8-to-os-2-values.yaml

```diff
@@ -0,0 +1,201 @@
+conditionalPackageInstalls:
```

Comment:
This seems like a lot of copy pasta configuration, do we have a task to consolidate this?  Not a blocker for this change, but I'm not sure where we are converging here